client/src/backend/routes.js

```diff
@@ -424,6 +424,13 @@ const routes = {
                 }
               ]
             },
+            {
+              name: "backendProjectExports",
+              exact: false,
+              component: "ProjectExport",
+              path: "/backend/projects/:pId/exports",
```

Comment:
The route here is OK and does not need to be changed to project-exportations. That's because this is part of the user interface, and it's OK for users to think of these as "exports". But, in code, which users do not see, we want to use names that are close to the model/resource names. Subtle, but important distinction.